NitroxServer/Communication/NetworkingLayer/NitroxServer.cs

```diff
@@ -64,7 +64,7 @@ protected void ProcessIncomingData(NitroxConnection connection, Packet packet)
             }
             catch (Exception ex)
             {
-                Log.Error("Exception while processing packet: " + packet + " " + ex);
+                Log.Error(ex, $"Exception while processing packet: {packet}");
```

Comment:
Do we not want to keep this for debugging if we get logs sent to us?

Reply:
Ignore my comment missed the exception still being logged, thought we had removed the logging for it.